modules/installation-dns-bare-metal.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-dns-bare-metal-{context}"]
+= Bare metal DNS requirements
+
+The following DNS records are required for an {product-title} cluster. In each
+record, `<cluster_name>` is the cluster name and `<base_domain>` is the cluster
+base domain that you specify in the `install-config.yaml` file.
+
+.Required DNS records
+[cols="1a,3a,5a",options="header"]
+|===
+
+|Component
+|Record
+|Description
+
+|Kubernetes API
+|`api.<cluster_name>.<base_domain>`
+|This DNS record must point to the load balancer
+for the control plane machines. This record must be resolvable by both clients
+external to the cluster and from all the nodes within the cluster.
+
+|Routes
+|`*.apps.<cluster_name>.<base_domain>`
+|A wildcard DNS record that points to the load balancer that targets the
+machines that  run the ingress router pods, which are the worker nodes by
+default. This record must be resolvable by both clients external to the cluster
+and from all the nodes within the cluster.
+
+.2+|Etcd
```

Comment:
@crawford had mentioned before that `etcd` is always lowercase.

Reply:
I've fixed this reference. The only reference in the docs that isn't lowercase now is to the `EtcdCluster` resource for the etcd Operator.